tests/Integration/Behaviour/Features/Context/Domain/CmsPageFeatureContext.php

```diff
@@ -0,0 +1,365 @@
+            $command->setIsIndexedForSearch((bool) $data['indexation']);
+        }
+        if (isset($data['active'])) {
+            $command->setIsDisplayed((bool) $data['active']);
+        }
+
+        $this->getCommandBus()->handle($command);
+        SharedStorage::getStorage()->set($reference, new \CMS($cmsId));
+    }
+
+    /**
+     * @When I attempt to edit cms page :reference providing illegal content value :value
+     */
+    public function editCmsPageProvidingIllegalContent($reference, $value)
+    {
+        $cmsId = (int) SharedStorage::getStorage()->get($reference)->id;
+        $command = new EditCmsPageCommand($cmsId);
+        $command->setLocalizedContent(
+            [$this->defaultLangId => $value]
+        );
+        try {
+            $this->getCommandBus()->handle($command);
+            SharedStorage::getStorage()->set($reference, new \CMS($cmsId));
+
+            throw new NoExceptionAlthoughExpectedException('Cms page was edited, but it was expected editing to fail');
+        } catch (\Exception $e) {
+            $this->latestResult = $e;
+        }
+    }
+
+    /**
+     * @When I delete cms page with id :id
+     */
+    public function deleteCmsPageById($id)
+    {
+        $command = new DeleteCmsPageCommand((int) $id);
+        $this->getCommandBus()->handle($command);
+    }
+
+    /**
+     * @Given cms page with id :id exists
+     */
+    public function assertCmsPageExistsById($id)
+    {
+        $query = new GetCmsPageForEditing((int) $id);
+        $this->getQueryBus()->handle($query);
+    }
+
+    /**
+     * @Given cms page with id :id should not exist
+     */
+    public function assertCmsPageDoesNotExistById($id)
+    {
+        $query = new GetCmsPageForEditing((int) $id);
+
+        try {
+            $this->getQueryBus()->handle($query);
+
+            throw new NoExceptionAlthoughExpectedException('Cms page exists. Expected it to be deleted');
+        } catch (CmsPageNotFoundException $e) {
+        }
+    }
+
+    /**
+     * @Then cms page :reference does not exist
+     */
+    public function assertCmsPageDoesNotExistByReference($reference)
+    {
+        try {
+            $cms = SharedStorage::getStorage()->get($reference);
+            $query = new GetCmsPageForEditing((int) $cms->id);
+            $this->getQueryBus()->handle($query);
+
+            throw new NoExceptionAlthoughExpectedException('Cms page exists. Expected it to not exist');
+        } catch (\Exception $e) {
+            if ($e instanceof NoExceptionAlthoughExpectedException) {
+                throw $e;
+            }
+        }
+    }
+
+    /**
+     * @Then /^cms page "(.*)" indexation for search engines should be (enabled|disabled)?$/
+     */
+    public function assertIndexation($reference, $status)
+    {
+        /** @var \CMS $cmsPage */
+        $cmsPage = SharedStorage::getStorage()->get($reference);
+        $statusBool = $status === 'enabled' ? true : false;
```

Comment:
`$statusBool = $status === 'enabled';` is enough. And please rename to `$statusBool` to `$isEnabled`.